app/components/css/about.module.css

```diff
@@ -1,223 +1,123 @@
-        gap: 40px;
-    }
-
-    .about_title{
-        font-size: 20px;
-        line-height: 22px;
-        text-align: center;
-    }
-    
-    .nav_button{
-        font-size: 16px;
-        line-height: 22.4px;
-    }
-    .about_2{
-        display: flex;
-        margin-top:100px;
-    gap: 24px;
-    align-items: inherit;
-    width: 279px;
-    /* justify-content: column; */
-    }
-    
-    .vector2{
-      left: 0;
-      bottom: 0;
-      top: 200px;
-      right: 200px
-    }
-
-    .vector1{
-       right: 100px
-    }
+.about_vector_ruby {
+    top: -5px;
+    right: 100px;
+}
+.about_vector_dart {
+    top: 620px;
+    left: 0;
 }
 
 
-@media (min-width: 501px) and (max-width: 1000px) {
-    .about{
-        display: grid;
-        gap: 500px;
-    }
-    .about_title{
-        text-align: center
-    }
-    .images_section{
-        margin-left: 155px ;
-        margin-top: 50px;
-    }
-    .about_2{
-        display: flex;
-        margin-top:100px;
-    gap: 24px;
-    align-items: inherit;
-    width: auto;
-    }
-
-    .vector2{
-        left: 0;
-        bottom: 0;
-        top: 200px;
-        right: 200px
-      }
-  
-      .vector1{
-         right: 100px
-      }
-}
 
-/* @media screen and (max-width: 1226px) {
+@media screen and (max-width: 1024px) {
     .about {
-        padding: 128px max(24px, 7%);
-    }
-}
-@media screen and (max-width: 1175px) {
-    .about {
-        padding: 128px max(59px, 6.5%);
-    }
-}
-@media screen and (max-width: 1100px) {
-    .about_cards {
-        flex-wrap: wrap;
+        display: flex;
+        flex-direction: column;
         justify-content: center;
-        gap: 32px 0;
     }
-}
-@media screen and (max-width: 980px) {
-    .about {
-        gap: 20px;
-        padding: 128px max(59px, 6.5%);
-    }
-}
-@media screen and (max-width: 500px) {
-    .about {
-        gap: 20px;
-        padding: 48px max(59px, 6.5%) 50px;
-        gap: 48px;
+    .about_content_ctn {
+        align-items: center;
+        gap: 24px;
     }
     .about_title {
-        font-size: 24px;
+        font-size: 1.75rem;
     }
-    .about_cards {
-        gap: 18px;
+    .about_nav {
+        width: 100%;
+        justify-content: space-between;
     }
-    .about_card {
-        padding: 24px;
-        gap: 24px;
+    .about_nav > button {
+        font-size: 1.25rem;
     }
-    .about_card img {
-        max-width: 48px;
-        width: 100%;
-        height: auto;
+    .paragraph {
+        font-size: 1rem;
+    }
+    .about_vector {
+        max-width: 130px;
+        max-height: 130px;
     }
-    .about_texts {
-        gap: 12px;
+    .about_vector_ruby {
+        top: 20px;
+        right: -57px;
     }
-    .about_sub_title {
-        font-size: 18px;
-        line-height: 131.25%;
+    .about_vector_dart {
+        top: 266px;
+        left: -30px;
     }
-    .about_paragraph {
-        font-size: 12px;
-        align-self: stretch;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
i set whitespace nowrap cause it is wrapping on small screens. so i set display flex for width for small screens.